<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="7652" started-at="2019-03-13T11:08:18Z" finished-at="2019-03-13T11:08:26Z">
    <groups>
    </groups>
    <test name="Avactis Admin Page Test" duration-ms="7652" started-at="2019-03-13T11:08:18Z" finished-at="2019-03-13T11:08:26Z">
      <class name="com.avactis.usertest.LoginTest">
        <test-method status="PASS" signature="extentSetUp()[pri:0, instance:com.avactis.usertest.LoginTest@4e7dc304]" name="extentSetUp" is-config="true" duration-ms="116" started-at="2019-03-13T11:08:18Z" finished-at="2019-03-13T11:08:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extentSetUp -->
        <test-method status="PASS" signature="setup(java.lang.String, java.lang.String)[pri:0, instance:com.avactis.usertest.LoginTest@4e7dc304]" name="setup" is-config="true" duration-ms="6990" started-at="2019-03-13T11:08:18Z" finished-at="2019-03-13T11:08:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[http://localhost/Avactis]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="verifyValidLogin()[pri:0, instance:com.avactis.usertest.LoginTest@4e7dc304]" name="verifyValidLogin" duration-ms="0" started-at="2019-03-13T11:08:25Z" data-provider="setLoginData" finished-at="2019-03-13T11:08:25Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at com.avactis.utilities.FileManager.getTestData(FileManager.java:64)
at com.avactis.testdata.UserDataProvider.setHospitalValidTestData(UserDataProvider.java:13)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyValidLogin -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.avactis.usertest.LoginTest@4e7dc304]" name="teardown" is-config="true" duration-ms="627" started-at="2019-03-13T11:08:25Z" finished-at="2019-03-13T11:08:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.avactis.usertest.LoginTest -->
    </test> <!-- Avactis Admin Page Test -->
  </suite> <!-- Suite -->
</testng-results>
